recipe/meta.yaml

```
@@ -7,11 +7,12 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  git_rev: db/scikit-build-core
+  git_url: https://github.com/TileDB-Inc/TileDB-Py.git
 
 build:
+  script: TILEDB_PATH=${PREFIX} {{ PYTHON }} -m pip install --no-build-isolation --no-deps --ignore-installed -v . # [not win]
+  script: set "TILEDB_PATH=${PREFIX}" && {{ PYTHON }} -m pip install --no-build-isolation --no-deps --ignore-installed -v . # [win]
```

```suggestion
  script: set "TILEDB_PATH=%LIBRARY_PREFIX%" && {{ PYTHON }} -m pip install --no-build-isolation --no-deps --ignore-installed -v . # [win]
```

The env var on Windows is `%LIBRARY_PREFIX%`. This is what was used previously in this feedstock (see above), and is what is used in [tiledb-feedstock](https://github.com/conda-forge/tiledb-feedstock/blob/f7f5e16ba8630a7fd074ef1a8b456b3a28935172/recipe/bld.bat#L13)